website/content/en/docs/getting-started/getting-started-with-karpenter/cloudformation.yaml

```diff
@@ -169,7 +169,7 @@ Resources:
                 "ec2:DescribeInstances",
                 "ec2:DescribeInstanceTypeOfferings",
                 "ec2:DescribeInstanceTypes",
-                "ec2:DescribeLaunchTemplates",
+                "ec2:DescribeLaunchTemplate*",
```

Comment:
why this change?

Reply:
Because a node template may have an existing launch template name set, we have to look up the data for the launch template via an AWS Api call to ec2:DescribeLaunchTemplateVersions

Launch template versions seems to be the only viable api to get at the full template data

To use this API, we need to add permission to describe launch template versions on the Karpenter controller role; I have update the docs to use ec2:DescribeLaunchTemplate* in the permissions set, instead of ec2:DescribeLaunchTemplates, in order to cover this additional permission (and any similar permissions required in future)

Reply:
@jonathan-innis oh sorry I wasn't aware of that proposal - sure lets not populate the label in the case where someone is using a custom launch template and sidestep this issue.